src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/PooledObjects/PooledArrayBuilder`1.cs

```diff
@@ -68,6 +70,16 @@ public void ClearAndFree()
         }
     }
 
+    private readonly ImmutableArray<T>.Builder CreateBuilder()
```

Comment:
Nit: Maybe call this `GetBuilder()`, since it doesn't really create a builder. It just gets one from the pool.